project/Build.scala

```diff
@@ -64,7 +64,7 @@ object Scrooge extends Build {
     version := libVersion,
     organization := "com.twitter",
     // 2.11-ification needs more work with mustache
```

Comment:
does this comment still apply? if so, can you add more color? otherwise, please rm.